server/routes/aapi.ts

```diff
@@ -39,10 +40,12 @@ router.get(
 			return;
 		}
 		const options = setOptions({
+			useOAuth: !!res.locals.identity.accessToken,
```

Comment:
silly question..
where is this res.locals.identity.accessToken set and what is it exactly?